application/utils/spreadsheet.py

```diff
@@ -237,3 +237,59 @@ def write_spreadsheet(title: str, docs: List[Dict[str, Any]], emails: List[str])
     for email in emails:
         sh.share(email, perm_type="user", role="writer")
     return "https://docs.google.com/spreadsheets/d/%s" % sh.id
+
+
+def generate_staggered_csv(database: db.Node_collection, docs: List[defs.CRE]) -> str:
+    maxOffset = 0
+    related = set()
+
+    def add_offset_cre(
+        cre: defs.CRE, database: db.Node_collection, offset: int, visited_cres: Set
+    ) -> List[Dict[str, str]]:
+        nonlocal maxOffset, related
+        maxOffset = max(maxOffset, offset)
+        rows = []
+
+        rows.append({f"CRE{offset}-ID": cre.id})
```

Comment:
We want to get rid of the -ID and do f"{id | name}|:cre.id, cre.name} instead of cre.id. Right?